santiao-guillen/homework-1/style.css

```diff
@@ -0,0 +1,226 @@
+*{
+    margin: 0px;
+    padding: 0px;
+}
+
+body{
+    margin: 20px 220px;
+}
+
+.backgraundImg{
+    position:relative;
```

Comment:
Make sure all the properties are alphabetically sorted

Reply:
I have a question, what do you mean by "all the properties are alphabetically sorted", i don't understand well